qemu/tests/cfg/block_hotplug.cfg

```diff
@@ -61,6 +61,7 @@
             reboot_method = system_reset
         - with_block_resize:
             only with_plug
+            only one_pci
```

Comment:
Hi aliang, Could paste the test result? thanks~

Reply:
@peixiu , test result has been pasted, please help to check, thank you.